README.rst

```diff
@@ -276,3 +275,23 @@ If the file is in spreadsheet format, the output directory will contain a *unfla
 **OpenaAg** rulesets check that the data contains the XML elements ``<opeang:tag>`` and ``<location>``, and that they include the right attributes expected for OpenAg data. Please read `OpenAg ruleset feature files <cove_iati/rulesets/iati_openag_ruleset/>`_ (written in `Gerkhin <https://github.com/cucumber/cucumber/wiki/Gherkin/>`_ style) for more information.
 
 **Org-ids** rulesets check that all organisation identifiers are prefixed with a registered `org-ids <http://org-id.guide>`_ prefix. Please read `Org-ids ruleset feature file <cove_iati/rulesets/iati_orgids_ruleset/>`_ for more information
+
+Development & Contributions
+===========================
+
+Development work on CoVE by Open Data Services is carried out in sprints. The issues for each sprint can be found at https://github.com/OpenDataServices/cove/projects . Other work is carried out from time to time. Outstanding issues for CoVE can be found at https://github.com/OpenDataServices/cove/issues . Please report any bugs!
+
+Community contributions to CoVE are welcome - please open an issue to discuss with the community before starting work.
+
+Credits
++++++++
+
+CoVE has been supported by open data standards organisations since its creation in 2015, as well as investment from Open Data Services. Key financial contributors have been:
+`Open Contracting Partnership <https://www.open-contracting.org/>`_
+`360Giving <http://www.threesixtygiving.org/>`_
+`Initiative For Open Ag Funding / InterAction <https://www.interaction.org/project/open-ag-funding/overview>`_
+
+As well as the team at Open Data Services, code contributions have been received from `@andylolz <https://github.com/andylolz>`_. Thanks! 
```

Comment:
> have been received from

Ummm… Not just received – they’ve been merged, in fact! [14 commits here](https://github.com/OpenDataServices/cove/commits?author=andylolz), [31 commits here](https://github.com/OpenDataServices/flatten-tool/commits?author=andylolz).

Reply:
@andylolz I'm not sure I follow - is there a way in which a contribution could be considered to be received if it's not merged? Surely in such a case a contribution would have been proposed and then rejected (or would still be pending)?

Reply:
You don’t propose a request – you send a request, and the other party receives a request (in English, that is. I don’t think any of this language has any special meaning in git/github).

Could you just remove the sentence, as requested by email? Thanks.